harvard-lts/jstor_aspace_publisher
# SS-260: implement sftp for lc and primo & SS-261

**Implement sftp for lc and primo.**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/SS-260

# What does this Pull Request do?
this implements sftp to primo andlc

# How should this be tested?

this has been deployed to dev and qa

1) run the integration test from dev or qa:

https://ltsds-cloud-dev-1.lib.harvard.edu:25003/integration
or
https://ltsds-cloud-qa-1.lib.harvard.edu:25003/integration

2) find the appropriate container running publisher in the cloud instance
3)` ls -l /tmp/JSTORFORUM/export/lc  /tmp/JSTORFORUM/export/primo` to see that the files were created
4) sftp dropbox@lc-control.lib.harvard.edu from inside the container 
5) `ls -l /data/dropbox/via-dev/incremental/ `  or `ls -l /data/dropbox/via-qa/incremental/ ` to confirm the files were successfully transfered

SS-261:
to confirm the headers are there, go to the container per the instructions above, and then inspect the xml export file in `/tmp/JSTORFORUM/export/lc`. you should see the headers.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? y

# Interested parties
Tag (@ mention) interested parties: @awoods
